<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отстой</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../creeper.jpg</normaloff>../../creeper.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>571</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="invariants">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Инварианты и тип кривой</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="ILabel">
        <property name="text">
         <string>I = ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="DLabel">
        <property name="text">
         <string>D = ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="BLabel">
        <property name="text">
         <string>B = ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="detLabel">
        <property name="text">
         <string>det = ?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="curveType">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="min">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="max">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="min_label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Минимум</string>
    </property>
   </widget>
   <widget class="QLabel" name="max_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Максимум</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_author"/>
    <addaction name="action_help"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_author">
   <property name="text">
    <string>Автор</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
